core/evidence/admission.py

```
@@ -0,0 +1,967 @@
+    validate_eval_event_type,
+    validate_idempotency_key,
+    validate_produced_at,
+    validate_validation_status,
+)
+from core.evidence.fingerprints import JsonScalar, JsonValue, fingerprint_payload
+from core.evidence.policies import (
+    normalize_upstream_ids,
+    validate_fingerprint,
+    validate_non_empty_token,
+)
+from core.evidence.promotion_ledger import (
```

Flake8 (7.3.0)

[error] 'core.evidence.promotion_ledger.ALLOWED_PROMOTION_DECISIONS' imported but unused

(F401)